backend.go

```diff
@@ -115,6 +115,14 @@ type kubeAuthBackend struct {
 
 	// tlsMu provides the lock for synchronizing updates to the tlsConfig.
 	tlsMu sync.RWMutex
+
+	// useAnnotationsAsAliasMetadata indicates that annotations, with the
+	// "vault.hashicorp.com/alias-metadata-" prefix, of the request JWT
+	// service account will be added to the alias metadata.
+	// Note Vault client or the token reviewer service account needs
+	// to be configured with the permission to get service account
+	// annotations from Kubernetes API
+	useAnnotationsAsAliasMetadata bool
```

Comment:
I think this should be a member of `kubeConfig` instead. The lifecycle of fields in `kubeAuthBackend` matches the lifecycle of the plugin process, but this value is tied to the most recent write to the config endpoint instead.

Reply:
Thank you for pointing this out!